connectors/spark/src/test/java/io/unitycatalog/connectors/spark/SchemaOperationsTest.java

```diff
@@ -0,0 +1,73 @@
+package io.unitycatalog.connectors.spark;
+
+import io.unitycatalog.server.utils.TestUtils;
+import org.apache.spark.sql.Row;
+import org.apache.spark.sql.SparkSession;
+import org.apache.spark.sql.catalyst.analysis.NoSuchNamespaceException;
+import org.junit.jupiter.api.Test;
+
+import static io.unitycatalog.server.utils.TestUtils.CATALOG_NAME;
+import static io.unitycatalog.server.utils.TestUtils.SCHEMA_NAME;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class SchemaOperationsTest extends BaseSparkIntegrationTest {
+  @Test
+  public void testCreateSchema() {
+    SparkSession session = createSparkSessionWithCatalogs(CATALOG_NAME, SPARK_CATALOG);
+    session.catalog().setCurrentCatalog(CATALOG_NAME);
+    session.sql("CREATE DATABASE my_test_database;");
+    assertTrue(session.catalog().databaseExists("my_test_database"));
+    session.sql(String.format("DROP DATABASE %s.my_test_database;", CATALOG_NAME));
+    assertFalse(session.catalog().databaseExists("my_test_database"));
+
+    session.catalog().setCurrentCatalog(SPARK_CATALOG);
+    session.sql("CREATE DATABASE my_test_database;");
+    assertTrue(session.catalog().databaseExists("my_test_database"));
+    session.sql(String.format("DROP DATABASE %s.my_test_database;", SPARK_CATALOG));
+    assertFalse(session.catalog().databaseExists("my_test_database"));
+    session.stop();
+  }
+
+  @Test
+  public void testSetCurrentDB() {
+    SparkSession session = createSparkSessionWithCatalogs(SPARK_CATALOG, TestUtils.CATALOG_NAME);
+    session.catalog().setCurrentCatalog(TestUtils.CATALOG_NAME);
+    session.catalog().setCurrentDatabase(SCHEMA_NAME);
+    session.catalog().setCurrentCatalog(SPARK_CATALOG);
+    // TODO: We need to apply a fix on Spark side to use v2 session catalog handle
+    // `setCurrentDatabase` when the catalog name is `spark_catalog`.
+    // session.catalog().setCurrentDatabase(SCHEMA_NAME);
+    session.stop();
+  }
+
+  @Test
+  public void testListSchema() {
+    SparkSession session = createSparkSessionWithCatalogs(SPARK_CATALOG);
+    Row row = session.sql("SHOW NAMESPACES").collectAsList().get(0);
```

Comment:
I'm wondering if we should rename all instances of NAMESPACE to SCHEMA for consistency. Which term is preferred in Unity Catalog? @cloud-fan

Reply:
In Unity Catalog we prefer SCHEMA. NAMESPACE is the Spark terminology.

Reply:
Let's use SCHEMA consistently.

Reply:
Ack. Done